packages/shared/src/hooks/useReadingHistoryContextMenu.ts

```diff
@@ -25,10 +26,7 @@ export default function useReadingHistoryContextMenu(): {
   ): void => {
     setReadingHistoryContextItem(readingHistory);
     setQueryIndexes(indexes);
-    const { right, bottom } = event.currentTarget.getBoundingClientRect();
-    showTagOptionsMenu(event, {
-      position: { x: right, y: bottom + 4 },
```

Comment:
What about the position removal here?

Reply:
the position arguments are being handled inside the `onMenuClick` function definition in the `useContextMenu` hook -- so this is simply removing duplicated code: https://github.com/dailydotdev/apps/blob/b3d1c8d32112083cdc6fd8c0a3220f8dd4a8e2fe/packages/shared/src/hooks/useContextMenu.ts#L41-L53